### PR TITLE
XDebug + docker user + symfony2 autocomplete

### PR DESCRIPTION
**Pull request by monte-fm**

### CODE REVIEW

File: configs/php5-fpm/xdebug.ini [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+xdebug.default_enable = 1
+xdebug.idekey = PHPSTORM
+xdebug.remote_enable = 1
+xdebug.remote_autostart = 1
+xdebug.remote_port = 9000
+xdebug.remote_handler=dbgp
+xdebug.remote_log=/var/log/xdebug/xdebug.log
+xdebug.remote_connect_back=1
+xdebug.max_nesting_level=250
+xdebug.remote_host = localhost
```
Comment (WhiteEagle88) [inline configs/php5-fpm/xdebug.ini]: Please, add new line

Comment (WhiteEagle88) [inline configs/php5-fpm/xdebug.ini]: You don't copy this file(

File: configs/php5-fpm/www.conf [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+listen.owner = www-data
+listen.group = www-data
+
+pm = dynamic
+pm.max_children = 5
+pm.start_servers = 2
+pm.min_spare_servers = 1
+pm.max_spare_servers = 3
+
+chdir = /
```
Comment (WhiteEagle88) [inline configs/php5-fpm/www.conf]: Please, add new line

**Comment by monte-fm** [comment]
added COPY of xdebug.ini
and added empty line in the end of file.

**Pull request merged**